dcos-log/api/v1/handlers.go

```diff
@@ -17,6 +17,15 @@ import (
 	"github.com/gorilla/mux"
 )
 
+type key int
+
+var streamKey key = 1
+
+func requestStreamKeyFromContext(ctx context.Context) bool {
```

Comment:
I guess the change is to move it from `routes.go` to `handlers.go`? Does it just logically make more sense here? If so, this would ideally have been its own commit with a commit summary explaining the rationale for the move.